EditContext/explainer.md

```diff
@@ -319,7 +365,7 @@ class EditableView {
 This [example application](edit_context_demo.html) shows how an author might build a simple editor that leverages the EditContext in a more holistic way.
 
 ## Interaction with Other Browser Editing Features
-By decoupling the view from text input, the EditContext gives up other DOM features that relate to editing.  An inventory of features related to editing in the web platform and their interaction with the EditContext follows:
+By decoupling the view from text input, the EditContext exposes other DOM features that relate to editing.  An inventory of features related to editing in the web platform and their interaction with the EditContext follows:

```

Comment:
exposes != gives up... maybe:
By decoupling the view from text input, the EditContext opts out of some editing behaviors that are currently only available through the DOM.  An inventory of those features and their interaction with the EditContext follows: